frontend/ui/elements/filemanager_menu_order.lua

```diff
@@ -136,7 +136,6 @@ local order = {
         "find_file",
         "----------------------------",
         "goodreads",
-        "opds_catalog",
```

Comment:
Please keep this. Sorting hint is just a backup. ;-)

Reply:
It seems to work fine without it. I was under the impression that sorting_hint was meant to replace hardcoding plugin menu entries. Am I wrong?

Reply:
Yes, definitely wrong. It only seems to work fine because it's literally doing the exact same thing in this one edge case. The menu is *designed*, not some random collection of items.

Reply:
NB There's a difference between plugins we ship and *user* plugins.

Reply:
So the purpose of the sorting hint for stock plugins is to be a backup in case the user removes the entry from default menu?

In that case the `coverimage` is missing its place on stock menus too :p

Reply:
> So the purpose of the sorting hint for stock plugins is to be a backup in case the user removes the entry from default menu?

I suppose, but as far as the intent goes it's closer to custom menu + new plugin, and of course trying to provide a good example to plugin authors (because most people just modify something like OPDS and not HelloWorld).

Reply:
> In that case the coverimage is missing its place on stock menus too :p

And that's definitely an issue, thanks.